AKostiv_Flask_App/app.py

```diff
@@ -0,0 +1,18 @@
+from flask import Flask, request, redirect, render_template
+
+app = Flask(__name__)
+app.config['SECRET_KEY'] = 'Thisisasecret!'
+
+
+@app.route('/')
+def index():
+    return render_template('index.html')
+
+@app.route('/greeting', methods = ['POST'])
+def result():
+    name = request.form['name']
```

Comment:
What happens when I press submit button without entering anything. You can add this as an enhacement

Reply:
@AKostiv8 you can look how others handled it, e.g. `https://github.com/fossasia/Flask_Simple_Form/pull/6/files` .